24th-App-Team-1-iOS/Feature/MessageFeature/Sources/Presentation/Components/ReservedMessageCountView.swift

```diff
@@ -0,0 +1,183 @@
+//
+//  ReservedMessageCountView.swift
+//  MessageFeature
+//
+//  Created by 최지철 on 12/20/24.
+//
+
+import UIKit
+
+import DesignSystem
+import Storage
+import MessageDomain
+import SnapKit
+import Then
+import Util
+
+final class ReservedMessageCountView: UIView {
+    
+    // MARK: - UI
+    
+    /// 타이틀(“지금 쪽지를 보낼 수 있어요” 등)
+    private lazy var titleLabel = WSLabel(wsFont: .Body01,
+                                          textAlignment: .left).then {
+        $0.textColor = .white
+        $0.text = "WeSpot" + String.MessageTexts.messageHomePostableTitleText
+    }
+    
+    /// 편지 이미지를 보여주는 이미지 뷰
+    private var messageLottieView = WSLottieView().then {
+        $0.lottieView.animation = DesignSystemAnimationAsset.bgMessageOpenAnimate.animation
+        $0.isStatus = true
+        $0.lottieView.loopMode = .playOnce
+    }
+    
+    /// "남은 시간"에 대한 소개 라벨
+    private let leftTimeIntoLabel = WSLabel(wsFont: .Body09,
+                                            text: String.MessageTexts.messageLeftTimeIntroText).then {
+        $0.textColor = DesignSystemAsset.Colors.gray300.color
+    }
+    
+    /// 타이머 아이콘
+    private let timerImage = UIImageView().then {
```

Comment:
```suggestion
    //setupViews
    timerImage.do  {
      $0.image = DesignSystemAsset.Images.icMessageHomeTimer.image
        $0.contentMode = .scaleAspectFit
    }
    
    // setupConstraints
    timerImage.snp.makeConstraints {
      $0.size.eqaulTo(20)
    }
```
- 각 메서드에 수행하도록 해주시면 감사하겠습니당 :)